go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
 	github.com/rjeczalik/notify v0.9.2
 	github.com/spf13/cobra v1.0.0
-	github.com/stretchr/testify v1.6.1 // indirect
+	github.com/stretchr/testify v1.6.1
```

Comment:
We've pretty much replaced all testify usage with https://pkg.go.dev/cdr.dev/slog@v1.3.0/sloggers/slogtest/assert

Reply:
Main difference is `NoError` -> `Success` and the string msg always comes first.

Reply:
There are some core issues with slog and I think we should try to move away from it.
testify is a widely used and well-supported lib, using it instead of a custom in-house one makes things easier to contribute for anyone within the company as well as from the community.